ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/vm/operations/SStoreOperation.java

```diff
@@ -27,8 +27,14 @@
 
 public class SStoreOperation extends AbstractOperation {
 
-  public SStoreOperation(final GasCalculator gasCalculator) {
+  public static final Gas FRONTIER_MINIMUM = Gas.ZERO;
+  public static final Gas EIP_1706_MINIMUM = Gas.of(2300);
+
+  private final Gas minumumGasRemaining;
+
+  public SStoreOperation(final GasCalculator gasCalculator, final Gas minumumGasRemaining) {
```

Comment:
(discussion) Since this new minimum gas check is backwards-compatible, should we just always apply it?  Rather than having separate minima for frontier vs istanbul?

Reply:
It is not backwards compatible.  Over 100 reference tests have broken so far in my test run and the build is not done yet.

Reply:
169 to be exact.  Across a broad set of tests.